src/java_bytecode/java_object_factory.cpp

```diff
@@ -460,8 +461,9 @@ class recursion_set_entryt
   /// Initialize a recursion-set entry owner operating on a given set.
   /// Initially it does not own any set entry.
   /// \param _recursion_set: set to operate on.
-  recursion_set_entryt(std::unordered_set<irep_idt, irep_id_hash> &_recursion_set):
-    recursion_set(_recursion_set)
+  explicit recursion_set_entryt(
```

Comment:
If not already, linter cleanups like this that fix existing code formatting problems should be in their own commit away from the functional changes.